sql/auto_uninstall.sql

```diff
@@ -1,2 +1,25 @@
-DROP TABLE IF EXISTS `membersonlyevent_event_membership_type`;
+-- +--------------------------------------------------------------------+
+-- | Copyright CiviCRM LLC. All rights reserved.                        |
+-- |                                                                    |
+-- | This work is published under the GNU AGPLv3 license with some      |
+-- | permitted exceptions and without any warranty. For full license    |
+-- | and copyright information, see https://civicrm.org/licensing       |
+-- +--------------------------------------------------------------------+
+--
+-- Generated from drop.tpl
+-- DO NOT EDIT.  Generated by CRM_Core_CodeGen
+--
+-- /*******************************************************
+-- *
+-- * Clean up the exisiting tables
+-- *
+-- *******************************************************/
+
+SET FOREIGN_KEY_CHECKS=0;
+
+DROP TABLE IF EXISTS `membersonlyevent_non_member_price_field_value`;
```

Comment:
Can we follow the table name as the existing table name for consistency?  for example, `membersonlyevent_event_*` for value that are saving form member only event tab?